### PR TITLE
[PDI-19849] - Regression in 9.4, Kitchen/Carte warning message "org.pentaho.kettle.repository.locator.impl.spoon.SpoonRepositoryProvider getRepository" appears in 9.4 kitchen/carte log

### PR DESCRIPTION
**Pull request by andreramos89**
Changed the java.util.logging.Logger to org.slf4j.Logger. This is the standard for PDI and as such, no longer shows the unwanted logs.

@smmribeiro 
@renato-s 
@bcostahitachivantara

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 9m 15s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
plugins/repository-locator/impl/spoon
```
---
:ok_hand: All tests passed!
 
Tests run: **2**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9045/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 31s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
plugins/repository-locator/impl/spoon
```
---
:ok_hand: All tests passed!
 
Tests run: **2**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9045/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: plugins/repository-locator/impl/spoon/src/main/java/org/pentaho/kettle/repository/locator/impl/spoon/SpoonRepositoryProvider.java [hunk 1]
```diff
@@ -20,19 +20,19 @@
...
-import java.util.logging.Logger;
 
 /**
  * Created by bryan on 4/15/16.
  */
 @ServiceProvider( id = "SpoonRepositoryProviderProvider",
   description = "Get the repository from the provided PentahoSession", provides = KettleRepositoryProvider.class )
 public class SpoonRepositoryProvider implements KettleRepositoryProvider, ServiceProviderInterface<KettleRepositoryProvider> {
-  private static Logger log = Logger.getLogger( SpoonRepositoryProvider.class.getName() );
+  private Logger log = LoggerFactory.getLogger( SpoonRepositoryProvider.class.getName() );
```
Comment (bcostahitachivantara) [inline plugins/repository-locator/impl/spoon/src/main/java/org/pentaho/kettle/repository/locator/impl/spoon/SpoonRepositoryProvider.java:35]: The common standard to declare Logger should be **private static final** Logger. Could exist some exceptions to this if you want to use it through inheritance, but does not seem to be the case.

**Review by bcostahitachivantara — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 8m 22s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
plugins/repository-locator/impl/spoon
```
---
:ok_hand: All tests passed!
 
Tests run: **2**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9045/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**